bidi-scanner/cli/cli.mjs

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Note the `mjs` extension: this is an EcmaScript module rather  than a CommonJS script.